src/utils/logger.js

```diff
@@ -0,0 +1,22 @@
+/* eslint no-console: "off" */
+
+function info(...args) {
+  if (typeof console !== "undefined") {
```

Comment:
just silly me wanna understand - why console would be undefined 🙀 ?

Reply:
Some browsers, though only IE comes to mind, didn't always support console.
But you are correct, it seems to be supported everywhere nowadays.

https://developer.mozilla.org/en-US/docs/Web/API/console